x/proof/module/query_claim_test.go

```diff
@@ -136,6 +136,10 @@ func TestClaim_List(t *testing.T) {
 	numClaimsPerSession := sharedtypes.DefaultNumBlocksPerSession
 	totalClaims := numSessions * numClaimsPerSession
 
+	// TODO_FLAKY: Ths following line in this test is flaky because test configuration
```

Comment:
Would love to look into this :monocle_face:

Reply:
Assigned your name to it!